### PR TITLE
Pipeline release v2.0

### PR DESCRIPTION
**Pull request by Vlad-Dembrovskyi**
#### Improvements:
 - Adds saving of all the process .command* log files to results/process-logs folder (#251)
 - Adds pipeline workdir `--cleanup` option to clear all intermediate files on pipeline successful completion (true by default, false for CloudOS) (#238, #284, [089d6e3](https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/pull/245/commits/3b71e038b186bb2bc92debacb02aede7b5dae917))
 - Adds pipeline `--error_strategy` parameter to be able to specify pipeline error strategy directly from command line (doesn't work if specified in config linked by `-c` or `-config` nextflow params) (#267)
 - Parametrizes google executor parameters so that pipeline can now be run on different CloudOS environments (#281)
 - Adds a new `--download_from` option `FTP` mode to download SRA samples from [EBI FTP](https://ftp.sra.ebi.ac.uk/vol1/fastq/) (#283)
- Adds new parameter `-save_unmapped` that makes saving of STAR unmapped files optional (false by default) (#284)

#### Fixes:
 - Adds missing trimmomatic logs to the multiqc report (#244)
 - Implemented correct support for input strandness in star process when `--stranded` is `second-strand` (was hardcoded to `strType=2` and only supported `first-strand` or `false` before) (#264)
 - Issue that stringti_merged results folder as well as some other folders are missing all or some files (#263)
 - Fix pipeline crash when `params.stranded` was set to `false` (#276)
 - Fixes old parameters in google.config that were undesirably overwriting nextflow.config parameters on CloudOS (#281, [217e202](https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/pull/245/commits/217e202cab3264c9d2d4cafe80b2476a2d837a85))
 
#### Updates:
 - Updates the following tools: (#248)
   - **STAR** `2.7.3` -> `2.7.9a` NOTE: Requires a new index! (updated in test profile)
   - **Samtools** `1.10` -> `1.13`
   - **StringTie** `2.1.3b` -> `2.1.7`
   - **Gffread** `0.11.7` -> `0.12.7`
   - multiqc `1.8` -> `1.11`
   - deeptools `3.4.0` -> `3.5.1`
   - bioconductor-rtracklayer `1.46.0` -> `1.52.0`
   - gffcompare `0.11.2` -> `0.12.6`
   - bedtools `2.29.2` -> `2.30.0`
   - sra-tools `2.10.8` -> `2.11.0`
   - pigz `2.3.4` -> `2.6.0`
   - gdc-client `1.5.0` -> `1.6.1`
 - Moves all containers to https://hub.docker.com/u/anczukowlab

#### Maintenance:
 - Consideably reduces number of basic redundant CI tests by removing completely the `max_retries` matrix and `push` from `on: [push, pull_request]`
 - Adds CI test for sra-downloading pipeline pathway (only supported with docker profile for now) (#253)

 
## To test
Pull the repository
```
git clone https://github.com/TheJacksonLaboratory/splicing-pipelines-nf
cd splicing-pipelines-nf
git checkout dev
```
Run the test profile with docker
```
NXF_VER=20.01.0 nextflow run . -profile ultra_quick_test,docker
```
image

Tested on sumner (JAX HPC) and approved by @angarb here: [link](https://lifebit-biotech.slack.com/archives/D02B1UGL8JY/p1632766255012100?thread_ts=1632496920.004800&cid=D02B1UGL8JY)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,40 @@ Documentation about the pipeline, found in the [`docs/`](docs) directory:
...
+   - bedtools `2.29.2` -> `2.30.0`
+   - sra-tools `2.10.8` -> `2.11.0`
+   - pigz `2.3.4` -> `2.6.0`
+   - gdc-client `1.5.0` -> `1.6.1`
+ - Moves all containers to https://hub.docker.com/u/anczukowlab
+
+#### Maintenance:
+ - Consideably reduces number of basic redundant CI tests by removing completely the `max_retries` matrix and `push` from `on: [push, pull_request]`
+ - Adds CI test for sra-downloading pipeline pathway (only supported with docker profile for now)
+#### QOL:
```
Comment (cgpu) [inline README.md:110]:
```Suggestion
#### Enhancements:
```

Comment (cgpu) [reply README.md:110]:
```Suggestion
#### Quality of Life:
```

File: README.md [hunk 2]
```diff
@@ -76,6 +76,40 @@ Documentation about the pipeline, found in the [`docs/`](docs) directory:
...
     * [Running on Sumner](docs/run_on_sumner.md)
     * [Running on CloudOS](docs/run_on_cloudos.md)
+    * [Running locally](docs/run_locally.md)
 
 ## Pipeline DAG
-<img src="https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/blob/c23ee9552eb033dec087fb3b6fb01fe26716ce29/DAG.png" alt="splicing_pip_dag" align = "left" width="600"/>
+<img src="https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/blob/c23ee9552eb033dec087fb3b6fb01fe26716ce29/DAG.png" alt="splicing_pip_dag" align = "center" width="600"/>
+
+
+## Changelog
```
Comment (cgpu) [inline README.md:85]: changelog can be a separate file in the root of the repo named `changelog.md`.

Add also the field:

```
PR: https://github.com/TheJacksonLaboratory/splicing-pipelines-nf/pull/245
```
to track the PR that introduced the changes easily.

File: README.md (hunk 1 above)
Comment (Vlad-Dembrovskyi) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Vlad-Dembrovskyi) [reply README.md:85]: done

**Comment by Vlad-Dembrovskyi** [comment]
Latest test from Britanny on CloudOS shows we are good to go for now. GTEX issues and improvements will be addressed in next release.

https://cloudos.lifebit.ai/public/jobs/619bbaa4fb26cf01dcee8446

**Pull request merged**